Automattic/wp-calypso
# Bluehost on Plans: Add dummy bluehost plan definitions

## Proposed Changes

* Refer: pdvytD-tZ-p2
This PR adds the definitions for a Bluehost plan which we intend to display in the pricing grid. The Bluehost plan will only be displayed in the landing page pricing grid, but since landing pages and Calypso share the same endpoint `/plans`, we need to add these dummy definitions in Calypso for the plans page to not break.



## Testing Instructions



* Visit `/start/plans` and verify that the plans page loads correctly. Also verify that the plans page renders correctly in the logged-in Calypso plans page.
* Apply patch D137499-code, sandbox public-api.wordpress.com, and navigate to `/start/plans`. Verify that the plans step renders correctly.
